core/arch/arm/tee/cache.c

```diff
@@ -49,13 +49,15 @@ TEE_Result cache_operation(enum utee_cache_operation op, void *va, size_t len)
 
 	switch (op) {
 	case TEE_CACHEFLUSH:
+#ifdef CFG_PL310
```

Comment:
By the way, we currently do not define a `CFG_OUTER_CACHE` configuration directive, as only PL310 is supported as outer cache and `CFG_PL310` cover the case. I prefer to keep relying on `CFG_PL310,` rather than introducing an extra (and almost useless) `CFG_OUTER_CACHE.` Do you agree ?